app/views/declaration/total_number_of_items.scala.html

```diff
@@ -26,19 +29,117 @@
 @this(
     govukLayout: gdsMainTemplate,
     govukButton: GovukButton,
+    govukInput: GovukInput,
+    govukFieldset: GovukFieldset,
     govukDetails : GovukDetails,
     pageTitle: pageTitle,
     exportsInputText: exportsInputText,
     errorSummary: errorSummary,
     sectionHeader: sectionHeader,
     saveButtons: saveButtons,
     tariffExpander: tariffExpander,
+    link: link,
+    paragraphBody: paragraphBody,
+    bulletList: bulletList,
     formHelper: FormWithCSRF,
     appConfig: AppConfig
 )
 
 @(mode: Mode, form: Form[TotalNumberOfItems])(implicit request: JourneyRequest[_], messages: Messages)
 
+@input(field: Field, labelKey: String, inputClasses: String) = {
+    @govukInput(Input(
+        id = field.id,
+        name = field.name,
+        value = field.value,
+        label = Label(content = Text(messages(labelKey)), classes = "govuk-label"),
+        classes = s"$inputClasses ${if(field.hasErrors) "govuk-input--error"}"
+    ))
+}
+
+@totalAmountInvoicedField = @{form(totalAmountInvoiced)}
+@totalAmountInvoicedCurrencyField = @{form(totalAmountInvoicedCurrency)}
+@totalAmountInvoicedErrors = @{totalAmountInvoicedField.errors ++ totalAmountInvoicedCurrencyField.errors}
+
+@currencyCodeAndAmountContent = {
+
+    @paragraphBody(messages("declaration.totalAmountInvoiced.paragraph1.text",
+        link(
+            text = messages("declaration.totalAmountInvoiced.paragraph1.link.text"),
+            call = Call("GET", appConfig.currencyCodesForDataElement410),
+            target = "_blank"
+        )
+    ))
+
+    @paragraphBody(messages("declaration.totalAmountInvoiced.paragraph2.text",
+        link(
+            text = messages("declaration.totalAmountInvoiced.paragraph2.link.text"),
+            call = Call("GET", appConfig.exchangeRatesForCustoms),
+            target = "_blank"
+        )
+    ))
+
+    <span class="govuk-hint" id="totalAmountInvoiced-hint">
+        @Html(messages("declaration.totalAmountInvoiced.hint"))
+    </span>
+
+
+    @if(totalAmountInvoicedErrors.nonEmpty){
+        <span class="govuk-error-message">
```

Comment:
Just a confirmation. If `totalAmountInvoicedErrors` is not empty , will this `span` tag and the one at line 82 be shown inlined ? And, if yes, is this what we want?

Reply:
TBH I'm not sure. This pattern was taken from the existing /additional-documentation page which has been through a DAC assessment so I'm assuming the HTML structure is correct. :crossed_fingers: :slightly_smiling_face: